jpn/jpn.config

```diff
@@ -42,7 +45,7 @@ classify_integer_matcher_multiplier 10
 #classify_class_pruner_multiplier    15
 assume_fixed_pitch_char_segment     T
 chop_enable                         F
-tessedit_char_blacklist             �
+tessedit_char_blacklist             °
```

Comment:
@Shreeshrii, here you replaced the ISO-8859 code by UTF-8. I think a similar change is needed for jpn_vert, too, and also in langdata_lstm and in several traineddata files which still use the ISO-8859 encoding. Is that correct?